### PR TITLE
The hangover script

### PR DESCRIPTION
**Pull request by TheRakken**

### CODE REVIEW

File: hangover.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  'Food poisoning',
+  'Not feeling well'
+]
+
+client = TwilioRestClient(TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN)
+
+client.messages.create(
+    to=number_of_boss,
+    from=my_number,
+    body="Gonna work from home. " + random.choice(excuses)
```
Comment (akarl818) [inline hangover.py]: "Gonna work from home. Locked out" Noticed that in the original as well..

Comment (TheRakken) [inline hangover.py]: ¯\_(ツ)_/¯

**Comment by NARKOZ** [comment]
Please `chmod +x` and place the file under `python` directory.

File: hangover.py (hunk 1 above)
Comment (NARKOZ) [inline hangover.py]: :joy:

**Pull request closed without merging**